### PR TITLE
Enable assertions

### LINKED ISSUE #82: Enable assertions
Update `build.gradle` to enable assertions in the team repo.

### PR DESCRIPTION
**Pull request by wkxcass**
Fixes #82 

Updated `build.gradle` to enable assertions in the team repo.

### CODE REVIEW

**Review by k-hian — DISMISSED** [review]
LGTM

File: build.gradle [hunk 1]
```diff
@@ -69,4 +69,8 @@ shadowJar {
     archiveFileName = 'addressbook.jar'
```
Comment (k-hian) [inline build.gradle:69]: Should we change this to `researchroster.jar` as well?

**Review by k-hian — COMMENTED** [review]
Minor suggestion, otherwise LGTM.

File: build.gradle (hunk 1 above)
Comment (wkxcass) [reply build.gradle:69]: Nice catch! Changing right now~

**Pull request merged**